docs-src/cloud/support-create-account.md

```diff
@@ -0,0 +1,35 @@
+---
+id: support-create-account
+title: How to create a Zendesk account for Temporal Support
+sidebar_label: Zendesk account
+description: Create a Zendesk account for Temporal Support.
+tags:
+  - temporal cloud
+  - support
+  - how-to
+---
+
+Zendesk has a per organization setting to associate user emails based on the domain name.
+You must create a Zendesk account for Temporal Support with the same domain name as your Temporal Cloud account.
```

Comment:
In fact, there is no explicit step for Cloud users to "create a Zendesk account", it's part of the onboarding process carried out by DSE. Cloud users can simply follow the process below to login to the Support portal by using the company email, through which Zendesk will give them access to their organization's tickets.

Reply:
Awesome. Fixed.